etc/error_code.toml

```diff
@@ -233,6 +233,11 @@ error = 'KV-Raft-RequestSnapshotDropped'
 description = ''
 workaround = ''
 
+[error.KV-Raft-ConfChangeError]
```

Comment:
I did not make changes to this file. I'm not sure why these lines were added :thinking:

Reply:
can you rebase your changes onto the latest master?

Reply:
Done!